required/tools/varioref.dtx

```diff
@@ -2264,7 +2264,7 @@
 %
 %  \begin{macro}{\vrefformat,\vrefdefaultformat}
 %    Japanese needs a different word order in \cs{vref@star} so this
-%    is separated out o that it can be changed on language level.
+%    is separated out so that it can be changed on language level.
```

Comment:
==>  . . . changed on the language . . .  

And would "at" be better than "on" ?

(Many similar cases.)

Reply:
I wouldn't know, but "at" sounds correct

Reply:
Again I was unclear!

The important thing is to use "the language level".

Since this is a technical term, it could be normal to use "on", but "at" sounds better to me.